CountingWords.rb

```diff
@@ -0,0 +1,6 @@
+freq = Hash.new(0)
+file = File.open("speech.txt")
+file.read.downcase.scan(/\b[a-z]{3,16}\b/){|word| freq[word] += 1}
+freq = freq.sort_by{|a, b| b}
+freq.reverse!
```

Comment:
.reverse made the counters go from 1 to 33. .reverse! allowed it to go from 33 to 1.